src/TcxWriter.cpp

```diff
@@ -77,21 +77,8 @@ void TcxWriter::onWorkout(const WorkoutInfo &i)  {
     writer.startElement("Activity");
     writer. writeAttribute("Sport", sport);
     writer. writeElement("Id", i.start_time.format());
-    writer. startElement("Lap");
-    writer. writeAttribute("StartTime", i.start_time.format());
-    writer. writeElement("TotalTimeSeconds", "0");
-    writer. writeElement("DistanceMeters", "0");
-      {
-        std::ostringstream ss;
-        ss<< (int)(i.calories/100);
-        writer.writeElement("Calories", ss.str());
```

Comment:
I didn't realise the checks weren't passing.  I've cleaned up the TcxWriter to generate valid XML.

Still not sure where to put the total calories for the activity though.  There really doesn't seem to be a good place to put them other than an extension to the schema?